examples/framework.py

```diff
@@ -116,7 +125,10 @@ def configure(self):
         if self.args.ask_password:
             password = getpass.getpass()
         else:
-            password = self.config.get("global", "password")
+            if self.config.has_section(self.g_jid):
+                password = self.config.get(self.g_jid, 'password')
```

Comment:
I don’t understand. As long as ``aioxmpp.structs.JID`` has ``__str__`` then it should not be a problem, should it? Or what else do you see there?